feat: add annotation queue section (#95)
Chainlit/literal-docs: concepts/evaluation/queue.mdx

---
title: "Annotation Queues"
---

> The Literal AI platform helps you organise and smoothen the human evaluation process by easily setting-up annotation queues.

Reviewer: I'm not sure smoothen is a word :grin: 

```suggestion
The Literal AI platform helps you organise and streamline the human evaluation process by easily setting-up annotation queues.
```

Each queue let you add data for review and provides an accessible and efficient ui for domain experts to score, tag and add to datasets crucial insights.

## Creating an annotation queue

> You can access your queues on the "Annotation Queues" page, and create them by pressing the "+" cta in the top right corner of the table.

Reviewer: IMO better to avoid jargon in public documentation.

```suggestion
You can access your queues on the "Annotation Queues" page, and create them by pressing the "+" button in the top right corner of the table.
```

> <img src="/images/create-annotation-queue.png" alt="Create annotation queue image" />

Reviewer: would you consider using [mintlify's frames](https://github.com/Chainlit/literal-docs/blob/main/concepts/observability/multimodal.mdx?plain=1#L21) ? they look good and allow you to add a caption to the image

Author: Looks cool didn't know about those.

This will open a modal form, that you can fill with a **name** to identify the annotation queue and a **description** to convey the queue meaning.

<img src="/images/create-annotation-queue-form.png" alt="Create annotation queue image form" />

> <Note>By default each project comes with a "Default" annotation queue.</Note>

Reviewer: ```suggestion
<Note>Each new project comes with a "Default" annotation queue.</Note>
```

## Populating a queue with data

> With your queues created you can manually add items to review.
> Accessing your run/generation logs, select them and insert them into the queue.

Reviewer: Slight inconsistency in the tenses

```suggestion
With your queues created you can manually add items to review by accessing your run/generation logs, selecting them and clicking "Add to Annotation Queue".
```

<img src="/images/add-generations-queue.png" alt="Add generations to the queue" />

To more precisely pick the relevant data you are able to select specific steps.

<img src="/images/add-step-queue.png" alt="Add steps to the queue" />

## Review queues

> On the "Annotation Queues" page, you can open a specific queue, allowing users to start reviewing added items.
> 
> A reviewer can score and tags providing you with human feedbacks.

Reviewer: ```suggestion
On the "Annotation Queues" page, a reviewer can open a specific queue and start reviewing added items. They can provide you with human feedback by scoring and tagging each item.
```

<img src="/images/annotation-item-view.png" alt="The annotation item page" />

> A reviewer can also add a specific step data to an existing dataset.
> 
> Finally, once the work is done, the item can be marked as reviewed.

Reviewer: In general, passive voice weakens the writing by omitting the subject. "X can do Y to Z" is considered stronger than "Z can be Yed".

```suggestion
A reviewer can also add a specific step to an existing dataset for further processing.

Finally, once the work is done, they can mark the item as Reviewed to remove it from the Annotation Queue.
```

Author: This will not remove from the queue, it will keep the item but marked it as reviewed

Reviewer: my bad, updated the suggestion

## More granular control

> To keep track of what is happening in the queues, we also provide for super users a proper list of the items in the queues.

Reviewer: (am i right that super users = Admin role ? better to keep the wording consistent in that case)

```suggestion
To keep track of what is happening in the queues, we also provide a list of Annotation Queue items for Admin users.
```

Author: Not really, super users defined non domain-experts roles. Because we do not follow conventional roles I can not say "AI Engineer" or +, because this semantically does not make sense.

Reviewer: i see, suggestion updated !
